pydash/predicates.py

```diff
@@ -77,6 +78,34 @@
 RegExp = type(re.compile(''))
 
 
+def eq(value, other):
+    """Checks if :attr:`value` is equal to :attr:`other`.
+
+    Args:
+        value (mixed): Value to compare.
+        other (mixed): Other value to compare.
+
+    Returns:
+        bool: Whether :attr:`value` is equal to :attr:`other`.
+
+    Example:
+
+        >>> eq({'a': 1}, {'a': 1})
+        True
+        >>> eq(None, None)
+        True
+        >>> eq(None, '')
+        False
+        >>> eq('a', 'a')
+        True
+        >>> eq(1, str(1))
+        False
+
+    .. versionadded:: TODO
+    """
+    return is_equal(value, other)
```

Comment:
Lodash's `eq` function does strict equality comparison: https://github.com/lodash/lodash/blob/1665876a1762d57987246354c4e8d61034aa1f20/lodash.js#L11198

So you should do the comparison with `is` instead of using `is_equal()`.